NEWS.md

```diff
@@ -7,5 +7,4 @@
 * Change argument of `fit_setting_contacts` from `survey_population` to just `population`
 * Reduce vignette size, closes [#28](https://github.com/njtierney/conmat/issues/28)
 * Add transmission probability data from Eyre et al., closes [#39](https://github.com/njtierney/conmat/issues/39)
-* Add setting weights, related to [#44](https://github.com/njtierney/conmat/issues/44)
```

Comment:
I don't think you need to remove this now, since the setting weights are added, just not from Eyre, right? So you can stay something like, "Added setting weights, related to [#44](https://github.com/njtierney/conmat/issues/44) (but no longer Eyre weights)